atom/browser/api/atom_api_window.cc

```diff
@@ -816,6 +832,9 @@ void Initialize(v8::Local<v8::Object> exports, v8::Local<v8::Value> unused,
                            &mate::TrackableObject<Window>::FromWeakMapID);
   browser_window.SetMethod("getAllWindows",
                            &mate::TrackableObject<Window>::GetAll);
+  browser_window.SetMethod("_setDeprecatedOptionsCheck",
```

Comment:
I prefer to make it a method of `exports` instead of `BrowserWindow`, so users won't be able to access it.

```js
const {_setDeprecatedOptionsCheck} = process.atomBinding('window');
```